jerry-core/ecma/builtin-objects/ecma-builtin-helpers-json.c

```diff
@@ -19,15 +19,15 @@
 
 #include "lit-char-helpers.h"
 
-#if JERRY_BUILTIN_JSON
-
 /** \addtogroup ecma ECMA
  * @{
  *
  * \addtogroup ecmabuiltinhelpers ECMA builtin helper operations
  * @{
  */
 
+#if JERRY_BUILTIN_JSON
+
```

Comment:
I feel this change (or these changes) somewhat arbitrary. There are lots of such C files, e.g., in the builtin-objects directory, that use the `#if-@{-@}-#endif` structure. This file used the `#if-@{-#endif-@}` pattern, which of course needs to be fixed, but I am not sure whether the `@{-#if-#endif-@}` structure is the way to go. But if we go that way, we should apply it consistently throughout the whole codebase.